src/cron.ts

```diff
@@ -6,23 +6,23 @@ import * as ogAccess from "./api/ogs";
 const jobs = [
   {
     desc: "Retrieve Strikes",
-    tab: "50 * * * *",
+    tab: "50 1 * * *",
     job: (): void => {
       console.log("Retrieving Strikes");
       strikeAccess.retrieveStrikes();
     }
   },
   {
     desc: "Retrieve Meetings",
-    tab: "45 * * * *",
+    tab: "45 1 * * *",
```

Comment:
das heißt plenumsdaten aktualisieren wir nur noch einmal am tag?
warum?

Reply:
weil der abruf der plenardaten und der streikdaten dafür sorgt, dass der benachrichtigungsstatus zurückgesetzt wird (im falle der plenardaten auf true, deswegn senden wir für die nie notifs, im falle der streikdaten auf false). Wenn wir jz stündlich den status zurücksetzen, hat das zu folge, dass stündlich eine notif gesendet wird. Das ganze ist ja auch schon teilweise mit TODO markiert und es wäre definitiv wichtig, daran zu arbeiten, dass die statusse nicht zurückgesetzt werden, aber bisschen out of scope für nen kleinen fix dachte ich